scripts/common.py

```diff
@@ -280,7 +280,17 @@ def open_new_tab_in_browser_if_possible(url):
         print('Please open the following link in browser: %s' % url)
         return
     browser_cmds = ['brave', 'chromium-browser', 'google-chrome', 'firefox']
-    for cmd in browser_cmds:
+    print(
+        'Please choose your default browser from the list using a number. '
+        'It will be given a preference over other available options.')
```

Comment:
```suggestion
    print(
        'Please choose your default browser from the list using a number. '
        'It will be given a preference over other available options.'
    )
```